src/components/not-found.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const NotFound = (props) => {
+    return (
```

Comment:
Right now we're using 2 spaces. Need to add prettier to the build process to catch formatting issues

Reply:
Got it, I'll fix that manually for now

Reply:
@nick Do you have the prettier configuration that you want somewhere? I can add it to the build process if you want

Reply:
We can use the one from origin-js:

https://github.com/OriginProtocol/origin-js/blob/master/package.json#L22-L25